flink-runtime/src/main/scala/org/apache/flink/runtime/jobmanager/JobManager.scala

```diff
@@ -149,6 +154,9 @@ class JobManager(
 
   var leaderSessionID: Option[UUID] = None
 
+  private var metricsRegistry : FlinkMetricRegistry = _
+  private var jobManagerMetricGroup : JobManagerMetricGroup = _
```

Comment:
`null` values are discouraged in Scala. Better use `Option[FlinkMetricRegistry]` which you initialize with `None`.